README.md

```diff
@@ -71,26 +69,14 @@ case class MyClass(
   option: Option[String])
 ```
 
-Second, define a configuration:
-
-`application.json`
-```json
-{ 
-  "boolean": true,
-  "port": 8080, 
-  "adt": { 
-    "type": "adtb", 
-    "b": 1 
-  }, 
-  "list": ["1", "20%"], 
-  "map": { "key": "value" } 
-}
-```
+Second, define an `application.conf` file like
+[this](https://github.com/pureconfig/pureconfig/blob/master/docs/src/main/resources/application.conf) and add it as a
+resource file of your application.
```

Comment:
Should we spell out what "add it as a resource file of your application" means in the context of common build tools? That is, usually it means to put the file in `src/main/resources`, right?

Reply:
With "a resource file of your application" I meant a [JVM resource](https://docs.oracle.com/javase/8/docs/technotes/guides/lang/resources.html), a concept agnostic of build tools. But it seems nice to specify where to put them when using SBT, the main Scala build tool.